flexget/plugins/filter/crossmatch.py

```diff
@@ -94,10 +99,31 @@ def entry_intersects(self, e1, e2, fields=None):
             log.trace('v1: %r' % v1)
             log.trace('v2: %r' % v2)
 
-            if v1 == v2:
-                common_fields.append(field)
+            if advanced == True:
+                v1Re = v1.lower()
+                v1Re = unicodedata.normalize('NFKD', v1Re).encode('ASCII', 'ignore')
+                v1Re = re.sub("[^a-zA-Z0-9 ]", " ", v1Re)
+                v1Re = v1Re.replace(' ','.*')
+                v1Re = '.*' + v1Re + '.*'
+                v2Re = v2.lower()
+                v2Re = unicodedata.normalize('NFKD', v2Re).encode('ASCII', 'ignore')
+                v2Re = re.sub("[^a-zA-Z0-9 ]", " ", v2Re)
+                v2Re = v2Re.replace(' ','.*')
+                v2Re = '.*' + v2Re + '.*'
```

Comment:
Idea:

mode: standard (default)
mode: flexible (new mode)

Leaves possibility to future modes.